requirements/base.txt

```diff
@@ -24,3 +24,6 @@ django-role-permissions==3.1.1 # https://github.com/vintasoftware/django-role-pe
 # Django REST Framework
 djangorestframework==3.12.4  # https://github.com/encode/django-rest-framework
 django-cors-headers==3.10.0 # https://github.com/adamchainz/django-cors-headers
+djangorestframework-simplejwt==5.0.0  # https://github.com/jazzband/djangorestframework-simplejwt
+# Yet another Swagger generator
+drf-yasg2==1.19.4  # https://github.com/JoelLefkowitz/drf-yasg
```

Comment:
we dont need this right?

Reply:
We will need it when we implement the Swagger-UI won't we?